ext/native/base/NKCodeFromSDL.h

```diff
@@ -46,7 +47,7 @@ static const std::map<int, int> KeyMapRawSDLtoNative = InitConstMap<int, int>
 	(SDLK_RIGHTBRACKET, NKCODE_RIGHT_BRACKET)
 	(SDLK_BACKSLASH, NKCODE_BACKSLASH)
 	(SDLK_SEMICOLON, NKCODE_SEMICOLON)
-	// (SDLK_, NKCODE_APOSTROPHE) // I cannot find the SDL equiv
+	(SDL_SCANCODE_0, NKCODE_APOSTROPHE)
```

Comment:
Okay, so I looked into this some.

SDL_SCANCODE_0 = 39 = ASCII `'`.  Correct value: `SDLK_QUOTE`.
SDL_SCANCODE_COPY = 124 = ASCII `|`.  There is no SDLK define.  Correct value: `'|'`.
SDL_SCANCODE_CUT = 123 = ASCII '{'.  There is no SDLK define.  Correct value: `'{'`.
SDL_SCANCODE_PASTE = 125 = ASCII '}'.  There is no SDLK define.  Correct value: `'}'`.
SDL_SCANCODE_KP_D = 191 = Extended ASCII '¿'.  There is no SDLK define.  Correct value: `191`.
SDL_SCANCODE_UNKNOWN = 0 = ASCII NUL.  Correct value: `SDLK_UNKNOWN`.

-[Unknown]